src/frontend/src/components/AuthorResult/AuthorResult.jsx

```diff
@@ -81,30 +78,23 @@ export default function AuthorResult() {
                   <ListItemIcon className={classes.icons}>
                     <PermContactCalendarIcon />
                   </ListItemIcon>
-                  <ListItemText className={classes.font} primary="Username" />
+                  <ListItemText className={classes.font} primary={`Username: ${author}`} />
                 </ListItem>
                 <ListItem button className={classes.infoLine}>
                   <ListItemIcon className={classes.icons}>
                     <EventIcon />
                   </ListItemIcon>
-                  <ListItemText className={classes.font} primary="Date of Last Post: " />
+                  <ListItemText
+                    className={classes.font}
+                    primary={`Date of Last Post: ${postDate}`}
```

Comment:
`postDate` should probably get converted to a human readable date.  You can probably use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat